### PR TITLE
Add Eerie Spell and G-Max Depletion activation messages

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/text.js [hunk 1]
```diff
@@ -468,6 +468,9 @@ exports.BattleText = {
 	dragonascent: {
 		megaNoItem: "  [TRAINER]'s fervent wish has reached [POKEMON]!",
 	},
+	eeriespell: {
+		activate: "  It reduced the PP of [TARGET]'s [MOVE] by [NUMBER]!",
```
Comment (Marty-D) [inline data/text.js:472]:
```suggestion
		activate: "#spite",
```

**Comment by Marty-D** [comment]
👍

**Comment by Zarel** [comment]
Wait, I never got around to writing a build script for this. I guess I should prioritize that.

**Comment by Zarel** [comment]
OK, done:

https://github.com/smogon/pokemon-showdown-client/commit/7fc87cad1c195a280898a6f684554a8f7aeb8738

In the future, edits will be done in the server's `data/text` files.

**Pull request merged**